frontend/src/components/User/UserProfile.tsx

```diff
@@ -136,6 +140,15 @@ const UserProfile = ({ id }: UserProfileProps) => {
                 <div>
                   <div className="text-lg mt-3 font-bold">{currentUser?.name}</div>
                   <p className="text-gray-400">{subscribers.length} Followers</p>
+                  {loadingBlogCount?"loading":(
+                    <p className="text-gray-400">{blogCount} Blogs</p>
+                  )}
+                  {loadingBookmarkCount?"loading":(
+                    <p className="text-gray-400">{bookMarkCount} BookMarks</p>
+                  )}
+                  {loadingMemberSince?"loading":(
+                    <p className="text-gray-400">Member : {member}</p>
+                  )}
```

Comment:
Looks like linting is not applied on your frontend code. 
Please run lint:fix, format npm commands inside frontend folder once. For reference, checkout package.json scripts

Reply:
when i do this lint:fix it modifies all the frontend files help me with this

Reply:
That's the intended behaviour. It will fix all frontend files that have formatting issues.

Reply:
You need to run the npm run format command as well after this